src/lib/int.ts

```diff
@@ -381,6 +381,14 @@ export class Int64 {
     return new Int64(this.value.neg());
   }
 
+  static add(x: Int64, y: Int64 | UInt32 | UInt64) {
+    return new Int64(x.value.add(y.value));
```

Comment:
can you delegate to the other add method here just to minimize code paths: ie.
`return new Int64(x.value).add(y.value)`